### ISSUE TITLE
[Bug] chi not visible but selection works

### ISSUE THREAD

**Issue by shawshankkumar** [Melkeydev/go-blueprint#155]
### What is the problem?

I tried creating a project with blueprint but chi as a framwork option was not visible.

### Operating System

macos

### Architecture Version (x86, x64, arm, etc)

x64

### Steps to reproduce

1. Install blueprint
2. run the command
3. voila, chi is not visible. 

Picture: 

<img width="1123" alt="image" src="https://github.com/Melkeydev/go-blueprint/assets/74819565/8bf4f409-8b12-4418-a1c1-6204a7ec7c4a">

### Relevant log output

```shell
> martin-scor          


  [ ] Gin
Features a martini-like API with performance that is up to 40 times faster thanks to httprouter

  [ ] Fiber
An Express inspired web framework built on top of Fasthttp

  [ ] Gorilla/Mux
Package gorilla/mux implements a request router and dispatcher for matching incoming requests to their respective handler

  [ ] HttpRouter
HttpRouter is a lightweight high performance HTTP request router for Go

  [ ] Echo
High performance, extensible, minimalist Go web framework

Press y to confirm choice.

> [x]  Mysql
 MySQL-Driver for Go's database/sql package

  [ ] Postgres
Go postgres driver for Go's database/sql package

  [ ] Sqlite
sqlite3 driver conforming to the built-in database/sql interface

  [ ] Mongo
The MongoDB supported driver for Go.

  [ ] None
Choose this option if you don't wish to install a specific database driver.

Press y to confirm choice.
                                                   
 Next steps cd into the newly created project with:
 • cd martin-scor
                 
 Tip: Repeat the equivalent Blueprint with the following non-interactive command:
 • go-blueprint create --name martin-scor --framework chi --driver mysql
```

**Comment by Ujstor** [comment]
On Linux and Windows, there is no such bug, but I will try to reproduce it on macOS.

**Comment by Ujstor** [comment]
We were unable to reproduce the bug. It seems that the problem is local. Please try reinstalling go-blueprint, and if that doesn't resolve the situation, install the latest version of Go

**Comment by shawshankkumar** [comment]
> We were unable to reproduce the bug. It seems that the problem is local. Please try reinstalling go-blueprint, and if that doesn't resolve the situation, install the latest version of Go

hey sure let me try again and i shall get back to you!

**Comment by shawshankkumar** [comment]
![image](https://github.com/Melkeydev/go-blueprint/assets/74819565/24dfce29-b035-44ed-ba9e-cda7dab49f3f)

works on a different machine of mine. Seems like an installation issue. Thanks @Ujstor , good stuff!
